packages/react/src/profiler.tsx

```diff
@@ -86,16 +82,16 @@ class Profiler extends React.Component<ProfilerProps> {
       const changedProps = Object.keys(updateProps).filter(k => updateProps[k] !== this.props.updateProps[k]);
       if (changedProps.length > 0) {
         const now = timestampInSeconds();
-        // eslint-disable-next-line deprecation/deprecation
-        this._updateSpan = this._mountSpan.startChild({
+        this._updateSpan = startInactiveSpan({
+          name: `<${this.props.name}>`,
+          op: REACT_UPDATE_OP,
+          origin: 'auto.ui.react.profiler',
+          startTimestamp: now,
           data: {
```

Comment:
l: while we at it, we should also rename this to `attributes` everywhere! We can also do this in a follow up, though.

Reply:
Done with c517e921a9dab31fdedcf3c80e17ccc79f53b0ac, will keep it in mind for the future.